buildtest/cli/__init__.py

```diff
@@ -313,6 +320,9 @@ def get_parent_parser():
         )
         return parent_parser
 
+    # Displays all hidden comands
+    if show_all_help:
+        help_all(subparsers)
```

Comment:
can you call this method at end of method so that these options show at bottom instead of top. Right now i see them as follows

```console
COMMANDS:
  
    tutorial-examples   Generate documentation examples for Buildtest Tutorial
    docs                Open buildtest docs in browser
    schemadocs          Open buildtest schema docs in browser
    unittests (test)    Run buildtest unit tests
    stylecheck (style)  Run buildtest style checks
    build (bd)          Build and Run test
    buildspec (bc)      Buildspec Interface
    config (cg)         Query buildtest configuration
```